FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by MIF50 on 15/09/2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+final class FeedImageMapper {
+	private struct Root: Decodable {
+		let items: [Item]
+		var feeds: [FeedImage] {
```

Comment:
In the context of the app a `feed` is a collection of feed images (`FeedImage`). Thus feeds should mean a collection of collections of feed images. Thus this property should be either `feed` or `images`

Please make the property name match with its type
```suggestion
		var feed: [FeedImage] {
```